taker/src/routes.rs

```diff
@@ -56,9 +57,37 @@ pub struct IdentityInfo {
     pub(crate) taker_peer_id: String,
 }
 
-#[rocket::get("/feed")]
-#[instrument(name = "GET /feed", skip_all)]
+#[derive(Debug, Copy, Clone, strum_macros::Display)]
+pub enum ContractSymbol {
+    BtcUsd,
+    EthUsd,
+}
+
+impl From<ContractSymbol> for model::ContractSymbol {
+    fn from(symbol: ContractSymbol) -> Self {
+        match symbol {
+            ContractSymbol::BtcUsd => model::ContractSymbol::BtcUsd,
+            ContractSymbol::EthUsd => model::ContractSymbol::BtcUsd,
```

Comment:
🔧 Should be `model::ContractSymbol::EthUsd`. Or `todo!()` if it doesn't exist yet, so that my brain doesn't explode.